text/src/swam/witx/parser/ImportContext.scala

```diff
@@ -0,0 +1,20 @@
+package swam.witx.parser
+
+import swam.witx.unresolved.BaseWitxType
+
+/**
+  *@author Javier Cabrera-Arteaga on 2020-03-18
+  */
+class ImportContext(val context: Map[String, BaseWitxType]) {
+
+  val types = context
+  def load(path: String) = {}
```

Comment:
Why is this method useful?

Reply:
It is not complete. The idea is to call it after a successful ```use``` parsing in the function definition, loading the types file, parsing it and updating the types map.